HISTORY.md

```diff
@@ -1,3 +1,14 @@
+unreleased
+==========
+
+  * deps: mocha@7.0.0
```

Comment:
Traditionally the history file is what the end consumers will see. The version of mocha we use internally doesn't change anything for the end user.

Reply:
Ok, I just added all of them.  I will remove all of the dev dep changes (which is most of them).

Reply:
Cool, no problem. The dev dep commits are typically pre-fixed with `build:` since they are part of the "build sub system" (from Node.js terms), and then by consequence all build stuff is just left out from the history file.

I tend to think of the history file's target audience is someone who uses this module directly and wants to know, at a quick glance, what the general list of changes that would matter to them when going from version X to Y.